### PR TITLE
Support for AIKONF7 FC

### PR DESCRIPTION
**Pull request by bhavpreet**
Adding support for AIKONF7 Flight controller.

Tested:
- all sensors work fine (GPS, Compass [HGLRC M8 Pro], Baro(External BME280))
- Multiple flights and few RTH.

### CODE REVIEW

**Comment by mmosca** [comment]
Please have a look at the new hardware policy:

https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md

File: src/main/target/AIKONF7/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+target_stm32f722xe(AIKONF7)
```
Comment (mmosca) [inline src/main/target/AIKONF7/CMakeLists.txt:1]: Can you add SKIP_RELEASES to this target?

**Comment by mmosca** [comment]
different pr merged

**Comment by bhavpreet** [comment]
Thanks!

**Pull request closed without merging**